### PR TITLE
Cleanup/remove unused header components

### PR DESCRIPTION
**Pull request by kingraphaii**
Related issues: [please specify]

## Description:

- removed the tailwind logo

- remove the "Dashboard" and "Team" link and added a link to "Teams".

- removed the notification bell icon from the top right

- removed the "settings" item from the menu

- removed the "profile" item from the menu

## Screenshots/videos

image

image

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have tested new or existing tests and made sure that they pass

### CODE REVIEW

File: backend/frontend/templates/frontend/partial_logged_in_main_header.html [hunk 1]
```diff
@@ -91,16 +58,10 @@
                   tabindex="-1"
                 >
                   <a
-                    href="#"
+                    href="{% url 'user_board' user.pk %}"
```
Comment (Ngoakor12) [inline backend/frontend/templates/frontend/partial_logged_in_main_header.html]: Your Board link is linking to the wrong user. It should link to the request user's board instead.

File: backend/frontend/templates/frontend/partial_logged_in_main_header.html [hunk 2]
```diff
@@ -11,53 +11,20 @@
         <div class="flex h-16 items-center justify-between px-4 sm:px-0">
           <div class="flex items-center">
             <div class="flex-shrink-0">
-              <img
-                class="h-8 w-8"
-                src="https://tailwindui.com/img/logos/mark.svg?color=indigo&shade=500"
-                alt="Your Company"
-              />
+              <span class="text-white">Tilde</span>
```
Comment (Ngoakor12) [inline backend/frontend/templates/frontend/partial_logged_in_main_header.html]: Bolder text to differentiate it from the other link?

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]

**Comment by Ngoakor12** [comment]
Also, the teams link should probably show that it's currently active when we're on `/users_and_teams_nav` page

image

**Comment by kingraphaii** [comment]
> Also, the teams link should probably show that it's currently active when we're on `/users_and_teams_nav` page

That would be nice, but holding this off for now as this was mainly meant to cleanup

**Comment by Ngoakor12** [comment]
The active page link is only highlighted for bigger screens, not for smaller screens

**Review by Ngoakor12 — APPROVED** [review]

**Pull request merged**